cluster-scope/overlays/nerc-ocp-infra/machineconfigs/configure-bond0/src/bond0.nmconnection

```diff
@@ -0,0 +1,17 @@
+[connection]
+id=bond0
+type=bond
+interface-name=bond0
+autoconnect=true
+connection.autoconnect-slaves=1
+
+[ethernet]
+mtu=9000
+
+[bond]
+mode=balance-rr
```

Comment:
Do we need to use this mode since the profiles are all using bond0 as the master?

Reply:
The `mode` parameter controls the configuration of the bond itself (whether using Etherchannel (which is `balance-rr`) or LACP (`mode=802.3ad`). See e.g. https://www.nodeum.io/howto/lacp-bonding-configuration-by-using-nmcli-centos-or-rhel-7.